app/locales/ja/translations.ts

```diff
@@ -471,13 +471,17 @@ export default {
         settings: 'Settings',
         comments: 'Comments',
     },
+    // These keys come from the "osf_status" cookie set by Flask to expose status messages added to the session with push_status_message()
```

Comment:
I think we should. They're more hassle than they're worth at this point, since they have translations for only a tiny (and shrinking) portion of the app.